script.js

```diff
@@ -0,0 +1,145 @@
+const addBox = document.querySelector(".add-box"),
+popupBox = document.querySelector(".popup-box"),
+popupTitle = popupBox.querySelector("header p"),
+closeIcon = popupBox.querySelector("header i"),
+titleTag = popupBox.querySelector("input"),
+descTag = popupBox.querySelector("textarea"),
+addBtn = document.querySelector(".button");
+
+
+var notes;
+if(localStorage.getItem("notes"))
+{
+    notes=JSON.parse(localStorage.getItem('notes'));
+}
+else{
+    notes=JSON.parse("[]");
+}
+let isUpdate = false, updateId;
+
+addBox.addEventListener("click", () => {
```

Comment:
Please define this function separately and then use it in "addEventListener" property.
Similarly make changes at all places.

Because by this, it will be more readable that exactly what will happen on addBox click